File: XKRW/Class/vc/Guide/Controller/login/XKRWForgetPasswordVC.m
//
//  XKRWForgetPasswordVC.m
//  XKRW
//
//  Created by Jiang Rui on 14-4-21.
//  Copyright (c) 2014年 XiKang. All rights reserved.
//

#import "XKRWForgetPasswordVC.h"

@interface XKRWForgetPasswordVC ()

@property (nonatomic,strong) UIWebView *webView;

@end

@implementation XKRWForgetPasswordVC

- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
{
    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
    if (self) {
        // Custom initialization
    }
    return self;
}

- (void)viewDidLoad
{
    [super viewDidLoad];
    [self addNaviBarBackButton];
    self.webView = [[UIWebView alloc]initWithFrame:CGRectMake(0, 0, XKAppWidth, XKAppHeight)];
    [self.view addSubview:_webView];
}

-(void)viewDidAppear:(BOOL)animated {
    [super viewDidAppear: animated];
    NSURL *url = [[NSURL alloc]initWithString:@"http://i.xikang.com/online/mobileusersystem/security/mobileresetpwd.html"];
    [_webView loadRequest:[NSURLRequest requestWithURL:url]];
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
{
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: XKRW/Class/vc/Guide/Controller/login/XKRWForgetPassWordVC.swift
//
//  XKRWForgetPassWordVC.swift
//  XKRW
//
//  Created by 忘、 on 15/7/2.
//  Copyright (c) 2015年 XiKang. All rights reserved.
//

import UIKit

class XKRWForgetPassWordVC: XKRWBaseVC {

    @IBOutlet weak var webView: UIWebView!
    override func viewDidLoad() {
        super.viewDidLoad()

        self.title = "忘记密码"
        self.addNaviBarBackButton()
        
        let url:NSURL = NSURL(string:"http://i.xikang.com/online/mobileusersystem/security/mobileresetpwd.html")!
        
        webView.loadRequest(NSURLRequest(URL: url))
        
        // Do any additional setup after loading the view.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
